docs/pangeo_forge_recipes/tutorials/index.md

```diff
@@ -20,6 +20,16 @@ xarray_zarr/opendap_subset_recipe
 
 These tutorials all use the {class}`pangeo_forge_recipes.recipes.HDFReferenceRecipe` class:
 
+Unlike the standard `XarrayZarrRecipe` recipe creation method, these reference recipes utilize [kerchunk](https://fsspec.github.io/kerchunk/) to create a ReferenceRecipe. The difference between these two methods is that the standard XarrayZarrRecipe creates a copy of the original dataset in the Zarr format, while the kerchunk-based ReferenceRecipe does not copy the data and instead creates a kerchunk mapping, which allows legacy formats such as (NetCDF, GRIB2, TIFF, FITS etc.) to be read as if they were Zarr stores. More details about how kerchunk works can be found in the [kerchunk docs](https://fsspec.github.io/kerchunk/detail.html). Kerchunk can be used independently outside of Pangeo-Forge. In these examples, Pangeo-Forge acts as a runner for kerchunk. Currently only HDF/NetCDF Reference Recipes are supported by Pangeo-Forge.
```

Comment:
```suggestion
Unlike the standard `XarrayZarrRecipe` recipe creation method, these reference recipes utilize [kerchunk](https://fsspec.github.io/kerchunk/) to create a `ReferenceRecipe`. The difference between these two methods is that the standard `XarrayZarrRecipe` creates a copy of the original dataset in the Zarr format, while the kerchunk-based `ReferenceRecipe` does not copy the data and instead creates a kerchunk mapping, which allows legacy formats such as (NetCDF, GRIB2, TIFF, FITS etc.) to be read as if they were Zarr stores. More details about how kerchunk works can be found in the [kerchunk docs](https://fsspec.github.io/kerchunk/detail.html). Kerchunk can be used independently outside of Pangeo-Forge. In these examples, Pangeo-Forge acts as a pipeline for generating kerchunk references. Currently only HDF/NetCDF Reference Recipes are supported by Pangeo-Forge.
```

Are you sure that GRIB is not supported?

Reply:
https://github.com/pangeo-forge/pangeo-forge-recipes/pull/438 ?

Reply:
"as if they were Zarr stores."

Something like:
"as if they were Zarr stores. This means that the users get the parallel chunk-wise access of a modern cloud-native library while reading the original data files in situ, in their original format".

Reply:
👍  to "dataset" over "store". 

We have no glossary. That would be useful though.

Reply:
Good call, updated!

Reply:
@rabernat , as far as I know, a grib2 reference recipe isn't supported yet. I chatted with Charles a bit ago about opening up a PR for a more generalized reference recipe class instead of reference recipe classes for each data type. The current WIP is here: https://github.com/pangeo-forge/pangeo-forge-recipes/pull/452/files.